stdlib/public/core/Misc.swift

```diff
@@ -169,6 +169,65 @@ func _rethrowsViaClosure(_ fn: () throws -> ()) rethrows {
   try fn()
 }
 
+/// A type whose values can be implicitly or explicitly copied.
+///
+/// Conforming to this protocol indicates that a type's value can be copied;
+/// this protocol doesn’t have any required methods or properties.
+/// You don't generally need to write an explicit conformance to `Copyable`.
+/// The following places implicitly include `Copyable` conformance:
+///
+/// * Structures declarations,
```

Comment:
should this read "structure declarations"?

Reply:
Yes, thanks — fixed in 587c79b6f1f.